control-plane/pkg/reconciler/consumergroup/evictor.go

```diff
@@ -0,0 +1,129 @@
+/*
+ * Copyright 2022 The Knative Authors
```

Comment:
was this updated? :smiley_cat:

Reply:
The boilerplate? no we didn't but the check is year-independent